cogeo_mosaic/backends/sqlite.py

```diff
@@ -0,0 +1,314 @@
+            rows = self.db.execute(
+                f"SELECT quadkey FROM {self.mosaic_name};",
+            ).fetchall()
+        return [r["quadkey"] for r in rows]
+
+    def write(self, overwrite: bool = False):
+        """Write mosaicjson document to an SQLite database.
+
+        Args:
+            overwrite (bool): delete old mosaic items in the Table.
+
+        Returns:
+            dict: dictionary with metadata constructed from the sceneid.
+
+        Raises:
+            MosaicExistsError: If mosaic already exists in the Table.
+
+        """
+        if self._mosaic_exists():
+            if not overwrite:
+                raise MosaicExistsError(
+                    f"'{self.mosaic_name}' Table already exists in {self.db_path}, use `overwrite=True`."
+                )
+            self.delete()
+
+        logger.debug(f"Creating '{self.mosaic_name}' Table in {self.db_path}.")
+        with self.db:
+            self.db.execute(
+                f"""
+                    CREATE TABLE IF NOT EXISTS {self._metadata_table}
+                    (
+                        mosaicjson TEXT NOT NULL,
+                        name TEXT NOT NULL,
+                        description TEXT,
+                        version TEXT NOT NULL,
+                        attribution TEXT,
+                        minzoom INTEGER NOT NULL,
+                        maxzoom INTEGER NOT NULL,
+                        quadkey_zoom INTEGER,
+                        bounds JSON NOT NULL,
+                        center JSON
+                    );
+                """
+            )
+            self.db.execute(
+                f"""
+                    CREATE TABLE {self.mosaic_name}
+                    (
+                        quadkey TEXT NOT NULL,
+                        assets JSON NOT NULL
+                    );
+                """
+            )
+
+        logger.debug(f"Adding items in '{self.mosaic_name}' Table.")
+        with self.db:
+            self.db.execute(
+                f"""
+                    INSERT INTO {self._metadata_table}
+                    (
+                        mosaicjson,
+                        name,
+                        description,
+                        version,
+                        attribution,
+                        minzoom,
+                        maxzoom,
+                        quadkey_zoom,
+                        bounds,
+                        center
+                    )
+                    VALUES
+                    (
+                        :mosaicjson,
+                        :name,
+                        :description,
+                        :version,
+                        :attribution,
+                        :minzoom,
+                        :maxzoom,
+                        :quadkey_zoom,
+                        :bounds,
+                        :center
+                    );
+                """,
+                self.metadata.dict(),
+            )
```

Comment:
This is a bit verbose, maybe there is a better way to write this

Reply:
👍 to named, not positional, arguments